.envrc

```diff
@@ -0,0 +1 @@
+layout_python3
```

Comment:
what is this?

Reply:
Oh I use [direnv](https://direnv.net/) for manage virtualenvs and this configures that. I'll gitignore it